cdc/capture_info.go

```diff
@@ -99,6 +101,10 @@ func newCaptureInfoWatch(
 		etcdWatchC := cli.Watch(ctx, captureEinfoKeyPrefix, clientv3.WithPrefix(), clientv3.WithRev(revision+1), clientv3.WithPrevKV())
 
 		for resp := range etcdWatchC {
+			failpoint.Inject("WatchCaptureInfoCompactionErr", func() {
+				watchResp <- &CaptureInfoWatchResp{Err: errors.Trace(mvcc.ErrCompacted)}
```

Comment:
Do we need to use `Trace` here?

Reply:
This simulates the common error handling code, which adds trace stack to the `resp.Err()`